doc/build.md

```diff
@@ -72,27 +74,28 @@ There are at least four arguments that must be passed to the script:
 | Flags                 | Description                                                                           |
 |--------------------   |-------------------------------------------------------------------------------------- |
 |   --maya-location     | directory where Maya is installed.                                                    |
-|  --pxrusd-location    | directory where Pixar USD Core is installed.                                          |
+|  --mayausd-location   | directory where MayaUSD is installed.                                                 |
 |  --devkit-location    | directory where Maya devkit is installed.                                             |
 | workspace_location    | directory where the project use as a workspace to build and install plugin/libraries  |
 
 Optional arguments are :
 | Flags                 | Description                                                                                         |
 |--------------------   |---------------------------------------------------------------------------------------------------- |
-|   --mayausd-location  | directory where MayaUSD is installed. By providing this, you will enable more features for          |
+|   --pxrausd-location  | directory where Pixar USD Core is installed.                                                        |
```

Comment:
--pxrausd-location  is the extra "a" on purpose ? And isn't this flag supposed to br removed now with mayausd being a requirement ?

Reply:
right, it's pxrusd-location, I assume this argument is still there (in build.py), to give user an opportunity in case they want to build against USD. Do you prefer not mentioning this argument at all?

Reply:
I think we need to keep it to build with maya 2024, right ?

Reply:
Yes, that's why I kept the building with Pixar USD section.

Reply:
Ok, so I think you only need to fix the typo --pxr**a**usd-location. Thanks.

Reply:
Sure, will fix that, @ppt-adsk are you ok with this?